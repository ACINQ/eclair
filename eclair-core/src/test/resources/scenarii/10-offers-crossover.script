# Offers which cross over still get resolved.
# Initial state: A=1000000 sat, B=1000000, both fee rates=10000 sat

A:offer 1000000,7b3d979ca8330a94fa7e9e1b466d8b99e0bcdea1ec90596c0dcc8d7ef6b4300c
A:commit
B:offer 2000000,6016bcc377c93692f2fe19fbad47eee6fb8f4cc98c56e935db5edb69806d84f6
B:recvoffer
B:recvcommit
B:commit

A:recvoffer
A:recvrevoke
A:recvcommit
B:recvrevoke

A:commit
B:recvcommit
A:recvrevoke

checksync
echo ***A***
A:dump
echo ***B***
B:dump
