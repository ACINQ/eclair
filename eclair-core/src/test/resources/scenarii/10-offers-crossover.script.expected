***A***
LOCAL COMMITS:
 Commit 1:
  Offered htlcs: (0,1000000)
  Received htlcs: (0,2000000)
  Balance us: 999000000
  Balance them: 998000000
  Fee rate: 10000
REMOTE COMMITS:
 Commit 2:
  Offered htlcs: (0,2000000)
  Received htlcs: (0,1000000)
  Balance us: 998000000
  Balance them: 999000000
  Fee rate: 10000
***B***
LOCAL COMMITS:
 Commit 2:
  Offered htlcs: (0,2000000)
  Received htlcs: (0,1000000)
  Balance us: 998000000
  Balance them: 999000000
  Fee rate: 10000
REMOTE COMMITS:
 Commit 1:
  Offered htlcs: (0,1000000)
  Received htlcs: (0,2000000)
  Balance us: 999000000
  Balance them: 998000000
  Fee rate: 10000
